include/treelite/tree.h

```diff
@@ -326,6 +382,8 @@ class Tree {
   inline void
   DeserializeTemplate(ScalarHandler scalar_handler, ArrayHandler array_handler);
 
+  friend class GTILBridge;  // bridge to enable optimized access to nodes from GTIL
```

Comment:
Can this be replaced by a simple getter?

Reply:
I'm still debating whether I should expose Node class to the end user, since it lacks information pertaining to categorical splits. (You need to query the tree object for that.)